resources/public/api/conf/1.0/application.yaml

```diff
@@ -1224,20 +1382,38 @@ paths:
                 _links:
                   self:
                     href: /customs/transits/movements/push-pull-notifications/box
-        '406':
-          description: Not Acceptable
+        '400':
+          description: Bad Request
           content:
             application/json:
               schema:
                 allOf:
-                  - $ref: '#/components/schemas/NotAcceptable'
-                  - description: The Accept header is missing or invalid.
+                  - $ref: '#/components/schemas/BadRequest1'
+                  - description: Client Id Required.
                     example:
-                      statusCode: 406
-                      message: The accept header is missing or invalid
+                      statusCode: 400
+                      message: Client Id Required
+                      code: CLIENT_ERROR
               example:
-                statusCode: 406
-                message: The accept header is missing or invalid
+                statusCode: 400
+                message: Client Id Required
+                code: CLIENT_ERROR
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/NotFound'
+                  - description: No box found for your client id
+                    example:
+                      statusCode: 404
+                      message: No box found for your client id
+                      code: CLIENT_ERROR
+              example:
+                statusCode: 404
+                message: No box found for your client id
+                code: CLIENT_ERROR

```

Comment:
In this block, `client id` is used but in preceding block `Client Id` is used. We need to be more consistent.